dist/scripts/macdeploy.sh

```diff
@@ -30,18 +30,24 @@ cd bin
 echo "Running macdeployqt"
 macdeployqt qView.app
 
-if [[ -f "qView.app/Contents/PlugIns/imageformats/kimg_heif.so" && -f "qView.app/Contents/PlugIns/imageformats/libqmacheif.dylib" ]]; then
+IMF_DIR=qView.app/Contents/PlugIns/imageformats
+if [[ (-f "$IMF_DIR/kimg_heif.dylib" || -f "$IMF_DIR/kimg_heif.so") && -f "$IMF_DIR/libqmacheif.dylib" ]]; then
     # Prefer kimageformats HEIF plugin for proper color space handling
     echo "Removing duplicate HEIF plugin"
-    rm "qView.app/Contents/PlugIns/imageformats/libqmacheif.dylib"
+    rm "$IMF_DIR/libqmacheif.dylib"
+fi
+if [[ (-f "$IMF_DIR/kimg_tga.dylib" || -f "$IMF_DIR/kimg_tga.so") && -f "$IMF_DIR/libqtga.dylib" ]]; then
```

Comment:
This was more of a pre-existing fix since I realized we were also shipping duplicate TGA plugins even prior to the updates.